src/index.html

```diff
@@ -1,14 +1,54 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="/src/images/logo.png"
+          alt="Moyo logo"
+          class="logo_img">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_item nav_item_1">
```

Comment:
Consider using the next structure for your nav: 
```
<ul>
  <li>
    <a href="..." ...> ... </a>
  </li>
  <li>
    <a href="..." ...> ... </a>
  </li>
</ul>
```
The child element of the list should be a list item, accordingly to the current HTML standard

Reply:
done